lib/slack.rb

```diff
@@ -1,11 +1,77 @@
-#!/usr/bin/env ruby
+require "dotenv"
+require "httparty"
+require "table_print"
 
-def main
-  puts "Welcome to the Ada Slack CLI!"
+require_relative "recipient"
+require_relative "user"
+require_relative "channel"
+require_relative "workspace"
 
-  # TODO project
+Dotenv.load
 
-  puts "Thank you for using the Ada Slack CLI"
+def main
+  workspace = Slack::Workspace.new 
+  puts "\n"
+  puts "Welcome to the Ada Slack CLI! This Slack workspace currently has #{workspace.users.count} users and #{workspace.channels.count} channels. Press enter to continue."
+  
+  user_input = gets.chomp
+  
+  until user_input == "quit"  
```

Comment:
I'd do this slightly differently. 
```suggestion
  tracking_int = 1
  until tracking_int == 0 
  
  case quit
    tracking_int = 0
  end
```

Reply:
I'd maybe make it a bool, and give it a better name, but this way you address the quit case separately.